src/field/pokemon.ts

```diff
@@ -1786,7 +1795,7 @@ export default abstract class Pokemon extends Phaser.GameObjects.Container {
     return (atkScore + defScore) * hpDiffRatio;
   }
 
-  getEvolution(): SpeciesFormEvolution | null {
+  getEvolution(): SpeciesFormEvolution | FusionSpeciesFormEvolution | null {
```

Comment:
```suggestion
  getEvolution(): SpeciesFormEvolution | null {
```

I'm not sure this change is needed since `FusionSpeciesFormEvolution` extends `SpeciesFormEvolution`